Makefile

```diff
@@ -170,23 +178,26 @@ vendor_libs: $(BACKBONE_OBJS) $(CLASSNAME_OBJS) $(LODASH_OBJS) $(REACT_OBJS) \
              $(BUILT)/test/vendor/mocha.js $(BUILT)/test/vendor/mocha.css \
              $(BUILT)/test/vendor/chai.js $(BUILT)/test/vendor/chai-as-promised.js
 
-$(BUILT)/add-on/chrome/content/preferences/prefs.js: $(VENV) add-on/preferences/prefs.js
+$(BUILT)/add-on/chrome/content/preferences/prefs.js: add-on/preferences/prefs.js $(VENV) Makefile
```

Comment:
Suggest using \ to split each dependency out to its own line for readability, here, and in the other instances of this change.